packages/css/src/plugins/atomicify-rules.tsx

```diff
@@ -69,7 +69,7 @@ const normalizeSelector = (selector: string | undefined) => {
  * @param parentClassName
  */
 const replaceNestingSelector = (selector: string, parentClassName: string) => {
-  return selector.replace(/ &/g, ` .${parentClassName}`);
```

Comment:
I had suspicion about this ` .${parentClassName}` spacing earlier. But I thought we had enough test cases to cover this. 😞   No worries. Lets add as many as test cases we can think of. What you say? 😃

Reply:
yeah definitely! i will make another test file